cypress/support/cypress-commands/lifecycle.js

```diff
@@ -554,3 +555,70 @@ Cypress.Commands.add('lifecycleSchemaChecks', (response, state) => {
   };
   cy.validationChecksForResponseAndSchemaResult(response, false, apiSchemaResult, false);
 });
+
+/**
+ * @module commands
+ * @function validateVisibilityState
+ * @description To validate app visibility for different lifecycle states
+ * @param {String} state - To check visibilityState of state
+ * @example
+ * validateVisibilityState('foreground');
+ */
+function validateVisibilityState(state) {
+  // Fetching the visibilityState for the states from env.
+  const visibilityState = Cypress.env(CONSTANTS.VISIBILITYSTATE);
+  if (visibilityState.hasOwnProperty(state)) {
+    // Get validation requirements for the current scenario from the moduleReqId JSON
+    const scenarioRequirement = UTILS.getEnvVariable(CONSTANTS.SCENARIO_REQUIREMENTS);
+
+    // Fetching the requirement IDs for the "visiblilityState" from the scenarioRequirement.
+    const lifecycleStateRequirementId = scenarioRequirement.find((req) =>
+      req.hasOwnProperty('visible_check')
```

Comment:
Let's add a "visible_check" constant if not already present

Reply:
Moved "visible_check" to constants.